src/OpenAI/Model/Gpt.php

```diff
@@ -40,6 +40,11 @@ public function supportsToolCalling(): bool
         return true;
     }
 
+    public function supportsImageInput(): bool
+    {
+        return $this->version->supportImageInput;
+    }
```

Comment:
The logic which belongs to image support should be extracted to a separate PR to avoid noise around the new chain processor feature

Reply:
True, will do so tomorrow